l10n/en/sub_navigation.ftl

```diff
@@ -24,12 +24,12 @@ sub-navigation-desktop-beta-and-developer = Desktop { -brand-name-beta } & { -br
 sub-navigation-desktop-nightly = Desktop { -brand-name-nightly }
 sub-navigation-features = Features
 sub-navigation-support = Support
-sub-navigation-addons = Addons
```

Comment:
BLOCKING Question: I was removing this string as its no longer used, but I have it set as the fallback. Do I need to leave it for now?

Reply:
Yeah, I think it needs to still be in the file to work as a fallback, and you can add a comment above it marking it as obsolete like:
`# Obsolete string`

Reply:
@craigcook Revised!